src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3181,6 +3181,12 @@ protected BoundExpression BindObjectCreationExpression(ObjectCreationExpressionS
         {
             var type = BindType(node.Type, diagnostics);
 
+            if (node.Type.Kind() == SyntaxKind.TupleType)
+            {
+                Debug.Assert(node.HasErrors, "new <tuple type> should be a syntax error");
+                return BadExpression(node, LookupResultKind.NotCreatable);
```

Comment:
Thanks for noticing this. Fixed.

---
In reply to: [81451427](https://github.com/dotnet/roslyn/pull/14161#discussion_r81451427) [](ancestors = 81451427,81443846)